src/routes/(console)/project-[project]/functions/function-[function]/deploymentDomains.svelte

```diff
@@ -8,31 +8,38 @@
 </script>
 
 <div class="u-flex u-gap-4 u-cross-center">
-    <a
-        href={`http://${domain.rules[0].domain}`}
-        target="_blank"
-        class="u-flex u-gap-4 u-cross-center">
-        <Trim alternativeTrim>
-            <span class="link">
-                {domain.rules[0].domain}
-            </span>
-        </Trim>
-        <span class="icon-external-link" aria-hidden="true" />
-    </a>
-    {#if domain.rules.length > 1}
-        <DropList bind:show={showDropdown} scrollable>
-            <Pill button on:click={() => (showDropdown = !showDropdown)}>
-                +{domain.rules.length - 1}
-            </Pill>
-            <svelte:fragment slot="list">
-                {#each domain.rules as rule, i}
-                    {#if i !== 0}
-                        <DropListLink href={`http://${rule.domain}`} external icon="external-link">
-                            {rule.domain}
-                        </DropListLink>
-                    {/if}
-                {/each}
-            </svelte:fragment>
-        </DropList>
+    {#if domain?.rules?.length > 0}
+        <a
+            href={`http://${domain.rules[0].domain}`}
+            target="_blank"
+            class="u-flex u-gap-4 u-cross-center">
+            <Trim alternativeTrim>
+                <span class="link">
+                    {domain.rules[0].domain}
+                </span>
+            </Trim>
+            <span class="icon-external-link" aria-hidden="true" />
+        </a>
+        {#if domain.rules.length > 1}
+            <DropList bind:show={showDropdown} scrollable>
+                <Pill button on:click={() => (showDropdown = !showDropdown)}>
+                    +{domain.rules.length - 1}
+                </Pill>
+                <svelte:fragment slot="list">
+                    {#each domain.rules as rule, i}
+                        {#if i !== 0}
+                            <DropListLink
+                                href={`http://${rule.domain}`}
+                                external
+                                icon="external-link">
+                                {rule.domain}
+                            </DropListLink>
+                        {/if}
+                    {/each}
+                </svelte:fragment>
+            </DropList>
+        {/if}
+    {:else}
+        <p>No rules available</p>
```

Comment:
Let's change this to domains as rules wouldn't make sense for most people.

```suggestion
        <p>No domains available</p>
```